extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -92,6 +92,14 @@ public class OidcTenantConfig extends OidcCommonConfig {
     @ConfigItem
     public Optional<String> endSessionPath = Optional.empty();
 
+    /**
+     * The paths this tenant applies to. Static tenants can be resolved based on path-matching patterns. Path-matching is
+     * done exactly in a same manner as is done for the 'quarkus.http.auth.permission.perm-1.paths' configuration property
+     * paths.
+     */
+    @ConfigItem
+    public Optional<List<String>> paths = Optional.empty();
```

Comment:
Should we name this property as `tenant-paths` just to be a bit more specific ? JavaDocs are clear what they mean, but at the configuration level, when the discovery is disabled, there could be up to 5 or so properties like `token-path`, `authorize-path`, and having `paths` alongside them might be a little be confusing

Reply:
The `tenant-paths` make sense. Thanks